### ISSUE TITLE
Creating an instance into a custom scenario fails to render

### ISSUE THREAD

**Issue by Razoric480** [godotengine/godot#77113]
### Godot version

3.5.stable

### System information

Windows 11 Pro

### Issue description

In a complex project, we are trying to render 3D elements using the visual server that is inside a viewport being used as a render target.

When rendering to the base viewport's scenario, no problem. But when we introduce a custom world and make the viewport use `own_world`, the objects refuse to appear even though they had their scenarios set to target this world.

### Steps to reproduce

Create a visual object using VisualServer using instance_create2 that targets a scenario on a viewport that uses `own_world`

### Minimal reproduction project

[OwnWorldIssue.zip](https://github.com/godotengine/godot/files/11482192/OwnWorldIssue.zip)

1. Run OwnWorldIssue.tscn scene, notice that there is 1 sphere and 1 triangle (mesh instances)
2. Disable `use_custom_world` exported variable on root control
3. Run scene, notice that there is 1 sphere and 2 triangles (2 mesh instances and 1 visual server artifact)

What is MeshInstance/base viewport doing that I'm not?

![image](https://github.com/godotengine/godot/assets/2823241/c994439d-ef2e-4ce2-b25d-db2f5861d1a8)
![image](https://github.com/godotengine/godot/assets/2823241/9f85943a-a817-47d0-901e-23f86a4e8113)

**Comment by kleonc** [comment]
That's not a bug, seems like the issue is a little unclear documentation. I also haven't instantly understood what the the docs meant. :upside_down_face:

TLDR: uncheck `own_world`.

Enabling `Viewport.own_world` will result in the Viewport duplicating the World set as its `world` property. E.g. if you'd have N Viewports, each with the same World set as `world`, and each having `own_world` enabled, then each such Viewport would end up with its own copy of that World. That's what *"will use a unique copy of the [World] defined in [member world]"* means.
https://github.com/godotengine/godot/blob/338114d47154298cc96bb3098653bb3db3943457/doc/classes/Viewport.xml#L246-L248

You can confirm it by checking that for Viewport with `own_world` enabled its `world` property and result of `find_world()` are different Worlds (`world` is the one which were copied, `find_world()` returns the created copy, the Viewport's own unique World).

If `own_world` is disabled and `world` is set, then `world` will be used by the Viewport as is, without copying it. Hence that's what you want.

So the issue was the mesh instance was being attached to a scenario not assigned to the given viewport.

**Comment by Razoric480** [comment]
![image](https://github.com/godotengine/godot/assets/2823241/ac096af7-0b0d-4b58-934c-0e688e77f5a3)
omg thank you

**Comment by Razoric480** [comment]
Closing, as it was a misunderstanding of what own_world does.
